tests/baselines/reference/declarationsAndAssignments.errors.txt

```diff
@@ -231,8 +237,14 @@ tests/cases/conformance/es6/destructuring/declarationsAndAssignments.ts(138,9):
         var a3: any[];
         var [...a] = [1, 2, 3];
         var [x, ...a] = [1, 2, 3];
+                   ~

```

Comment:
I'd change the variable names at these new errors in this test so that you're comparing with the expected type and there's no error.